### PR TITLE
`std::bits` fixes.

### PR DESCRIPTION
**Pull request by data-man**

### CODE REVIEW

File: lib/std/bits.c3 [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 /**
- * @require types::is_intlike($typeof(i)) `The input must be an integer or integer vector`
+ * @require types::is_intlike($typeof(self)) `The input must be an integer or integer vector`
  **/
-macro reverse(i) => $$bitreverse(i);
+macro reverse(self) => $$bitreverse(self);
 
 /**
- * @require types::is_intlike($typeof(i)) `The input must be an integer or integer vector`
+ * @require types::is_intlike($typeof(self)) `The input must be an integer or integer vector`
```
Comment (lerno) [inline lib/std/bits.c3:9]: "self" should not be used on non-methods

**Comment by lerno** [comment]
The bin_literal test needs to be updated to account for the change in variable name.

**Comment by data-man** [comment]
I removed unrelated code for literals.

**Comment by lerno** [comment]
I want to keep that code because it is still testing things that are not tested otherwise.

**Comment by data-man** [comment]
> things that are not tested otherwise

`rotr` and `rotl`?

**Comment by lerno** [comment]
Not just that, sometimes there are small changes that might be correct or not and sometimes a completely unrelated test will catch it.

**Comment by data-man** [comment]
Fixed now, I hope.

**Comment by lerno** [comment]
Thanks!

**Pull request merged**